### DISCUSSION TITLE
Add dedicated pagination component

### DISCUSSION THREAD

**Discussion by justenh** [livewire/flux#716] [Feature Requests]
It would be helpful if the pagination could be separated from the`<flux:table>` component.

**Comment by jeffchown** [comment]
@justenh That is actually already possible, you can show a paginator, via e.g.:
`<flux:pagination :paginator="$clients" />`

**Reply by lotje-kinable** [reply]
@jeffchown wow nice!

Do you know if there is a dedicated place to ask for updates in the docs?

**Reply by jeffchown** [reply]
@lotje-kinable I think you could @ Caleb and/or Josh in here and ask him to add it

**Reply by lotje-kinable** [reply]
@jeffchown will do! Thanks!

Hi @calebporzio  would you please add <flux:pagination :paginator="$orders" /> to
 [pagination docs](https://fluxui.dev/components/table#pagination) ? 
 
 I didn't know it even existed! Thanks in advance!
